src/fs/hardlinkDir.ts

```diff
@@ -9,24 +11,37 @@ export default async function hardlinkDir(existingDir: string, newDir: string) {
     dirs
       .map(async (relativePath: string) => {
         const existingPath = path.join(existingDir, relativePath)
-        const newPath = path.join(newDir, relativePath);
-        const stat = await fs.stat(existingPath)
+        const newPath = path.join(newDir, relativePath)
+        const stat = await fs.lstat(existingPath)
+        if (stat.isSymbolicLink()) {
```

Comment:
This can be combined with the 3rd statement, right? `stat.isSymbolicLink() || stat.isFile() `